### PR TITLE
[Enhancement][AutoBench] Adding more columns for detailed report 

### PR DESCRIPTION
**Pull request by prateekdesai04**
*Description of changes:*
This is an enhancement PR which gives out a more detailed report of all the benchmarks of Master vs v1.0.
Initially, the report had `Winrate`, `training time`, `inference time` and `rank`.
This may not provide sufficient information to determine if the framework is necessarily better hence this PR is to give out a more detailed report (upload the entire evaluation as a zipped artifact for every module).

P.S. We can add it as a 1.1 milestone item if required, may not be necessary.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CI/bench/evaluate.py [hunk 1]
```diff
@@ -188,17 +187,9 @@ def main():
...
-                        df1 = pd.read_csv(file_path, usecols=["time_train_s", "time_infer_s","rank"])
-
                     if file.startswith("AutoGluon") and file.endswith(".csv") and "pairwise" in root:
                         file_path = os.path.join(root, file)
-                        df2 = pd.read_csv(file_path, usecols=["framework", "Winrate"])
-                        df1 = df1.assign(Winrate=None, framework=None)
-                        df1["Winrate"] = df2["Winrate"]
-                        df1["framework"] = df2["framework"]
-                        df1 = df1.reindex(columns=["framework", "Winrate", "rank", "time_train_s", "time_infer_s"])
+                        df1 = pd.read_csv(file_path, usecols=["framework", "Winrate", "time_train_s", "time_infer_s","rank"])
```
Comment (prateekdesai04) [inline CI/bench/evaluate.py:192]: @Innixma I still keep this in case we have a request to remove some columns/files in the future.

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

File: CI/bench/evaluate.py (hunk 1 above)
Comment (Innixma) [reply CI/bench/evaluate.py:192]: Sounds good

**Review by Innixma — APPROVED** [review]
LGTM!

**Pull request merged**